pkg/openfeature/provider.go

```diff
@@ -34,7 +38,7 @@ type FlattenedContext map[string]interface{}
 type Reason string
 
 // FeatureProvider interface defines a set of functions that can be called in order to evaluate a flag.
-// vendors should implement
+// This should be implemented by vendors.
```

Comment:
```suggestion
// This should be implemented by flag management systems.
```

This term is more neutral. flagd isn't a vendor, for instance.

Reply:
Updated with the latest changeset